kolibri/plugins/device_management/assets/src/views/manage-content-page/channel-list-item.vue

```diff
@@ -183,6 +171,18 @@
           forExport: this.$route.query.for_export,
         });
       },
+      verticalPadding() {
+        return {
+          paddingBottom: `${this.windowGutter}px`,
+          paddingTop: `${this.windowGutter}px`,
+        };
```

Comment:
The window gutter width used in responsive grids is made available globally. Here, we use it to dynamically set vertical spacing in a way that's visually consistent with the current grid.